src/SourceGenerators/Uno.UI.SourceGenerators.Tests/XamlCodeGeneratorTests/Verifiers/CSGenerator.cs

```diff
@@ -2,7 +2,6 @@
 // Uncomment the following line to write expected files to disk
 // Don't commit this line uncommented.
 // #define WRITE_EXPECTED
-#define WRITE_EXPECTED
 #endif
 
 using System.Collections.Immutable;
```

Comment:
```suggestion
#if IS_CI && WRITE_EXPECTED
#error "WRITE_EXPECTED should not be defined!"
#endif

using System.Collections.Immutable;
```

Reply:
There is another `#error` few lines below. I think it's not correct, and the check using `IS_CI` is better.